CHANGELOG.md

```diff
@@ -1,3 +1,78 @@
+### citus v12.0.0 (July 11, 2023) ###
+
+* Adds support for schema-based sharding.
+  While `citus.enable_schema_based_sharding` GUC allows sharding the database
+  based on newly created schemas, `citus_schema_distribute()` allows doing so
+  for the existing schemas. Distributed schemas used for sharding the database
+  can be listed by using the view `citus_schemas`, monitored by using the view
+  `citus_stat_schemas`, and undistributed by using the udf 
+  `citus_schema_undistribute()`
+  (#6866, #6979, #6933, #6936 and many others)
+
+* Supports MERGE for non-colocated distributed tables and non-dist column joins (#6927)
```

Comment:
maybe:
Supports MERGE command across non-colocated distributed tables/subqueries, reference tables and joins on non-distribution columns